test/testcafe/spec/DeviceChallengePollViewOktaVerify_spec.js

```diff
@@ -1,82 +1,35 @@
-import { RequestLogger, RequestMock } from 'testcafe';
+import { RequestLogger, RequestMock, ClientFunction } from 'testcafe';
 import DeviceChallengePollPageObject from '../framework/page-objects/DeviceChallengePollPageObject';
-import IdentityPageObject from '../framework/page-objects/IdentityPageObject';
-import identify from '../../../playground/mocks/idp/idx/data/identify';
-import identifyWithDeviceProbingLoopback from '../../../playground/mocks/idp/idx/data/identify-with-device-probing-loopback';
-import loopbackChallengeNotReceived from '../../../playground/mocks/idp/idx/data/identify-with-device-probing-loopback-challenge-not-received';
 import identifyWithLaunchAuthenticator from '../../../playground/mocks/idp/idx/data/identify-with-device-launch-authenticator';
 
 const logger = RequestLogger(/introspect|probe|cancel|launch|launch-okta-verify|poll/);
 
+// const identifyWithLaunchAuthenticatorHttpCustomUri = Object.assign({}, identifyWithLaunchAuthenticator.default);
+const identifyWithLaunchAuthenticatorHttpCustomUri = JSON.parse(JSON.stringify(identifyWithLaunchAuthenticator));
+debugger;
+const mockHttpCustomUri = 'http://localhost:3000/launch-okta-verify';
+// replace custom URI with http URL so that we can mock and verify
+identifyWithLaunchAuthenticatorHttpCustomUri.authenticatorChallenge.value.href = mockHttpCustomUri;
```

Comment:
can you add a comment explaining, why we are setting the href here in the test and not in the mock file ?

Reply:
There's a comment on line 11. Replacing a custom URI with a HTTP URL so we can mock it and test it.